target/linux/generic/patches-4.4/308-mips32r2_tune.patch

```diff
@@ -1,17 +0,0 @@
-Add -mtune=34kc to MIPS CFLAGS when building for mips32r2
```

Comment:
It seems that MIPS 4KEc is supposed to implement MIPS32 R2. If this patch breaks it, please show me an example crash along with disassembly so that I can see which instruction it breaks on. Maybe there's a less intrusive tweak that we can do here.